### ISSUE TITLE
Meep /  IPython Animation Error

### ISSUE THREAD

**Issue by mucahitdemir** [NanoComp/meep#1004]
Hello!
I have been trying to run the first meep project uploaded on the website (https://nbviewer.jupyter.org/github/NanoComp/meep/blob/master/python/examples/straight-waveguide.ipynb)

I have received the results totally but when it comes to the animation part, the terminal always gives errors.
Moreover, I installed IPython. Could somebody help me? 
Best regards,

![Screen Shot 2019-08-29 at 10 54 39](https://user-images.githubusercontent.com/43001724/63921486-90127f00-ca4b-11e9-92f1-2dfcd296659b.png)

**Comment by smartalecH** [comment]
Do you also have Jupyter installed?

**Comment by mucahitdemir** [comment]
> Do you also have Jupyter installed?

Yes! Do I need to open or write some code for the Jupiter notebook to open it for creating a video?

**Comment by smartalecH** [comment]
If you just want to _save_ an animation, you don’t need Jupyter/iPython. If you want to _display_ your animation next to your code in an interactive environment, then you need to use the web interface. You can’t visualize an mp4 inside a shell. 

It’s hard to say why your ffmpeg pipe is breaking. It might just be because you are running the notebook in your shell. Try just running the interactive notebook in a web browser. Also, make sure you’re running the latest version of meep. There’s been a lot of bug fixes to the animation output.

**Comment by Nico2344** [comment]
Hello,

I installed MEEP last week on my Mac (10.14.6) and came out with the same issue. 

What seems to happen is that the code does not create the "media" folder and I suspect that the script crashes because the buffer is saturated (Not sure). Just changing the filename path to the current directory worked fine: 
`filename = "straight_waveguide.mp4"`

Another workaround when working with Jupyter notebooks (with google chrome) is using the method "to_jshtml" instead of "to_mp4":

`from IPython.display import HTML
HTML(Animate.to_jshtml(10).get_jshtml())`

It worked for me and I hope it will be the same for others.
Nico
